entgql/template/pagination.tmpl

```diff
@@ -505,5 +505,17 @@ var {{ $defaultOrder }} = &{{ $order }}{
 	},
 }
 
+// New{{ $edge }} return new {{ $edge }}
+func New{{ $edge }}(node *{{ $name }}, order *{{ $order }}) *{{ $edge }} {
```

Comment:
Let's add a `ToEdge` method to every node instead of having those global functions
```suggestion
func ({{ $r }} *{{ $name }}) ToEdge(order *{{ $order }}) *{{ $edge }} {
```